<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainGUI</class>
 <widget class="QMainWindow" name="MainGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="saveLabel">
        <property name="text">
         <string>Save as:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="csvButton">
          <property name="text">
           <string>CSV file</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="htmlButton">
          <property name="text">
           <string>HTML file</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="nameLabel">
          <property name="text">
           <string>Give a name for the file</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="nameLineEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
